### PR TITLE
FAQInstallation: update how verify signature

### PR DESCRIPTION
**Pull request by MarnixCroes**

### CODE REVIEW

File: docs/FAQ/FAQ-Installation.md [hunk 1]
```diff
@@ -43,8 +43,8 @@ This protects you against malicious man in the middle attacks where bad guys giv
 ### How can I verify PGP signatures?
 
 On the [WasabiWallet.io](https://wasabiwallet.io) website you can download the packages of the latest release.
-Ensure that you also download the separate signature `.asc` file.
-In the terminal, change the directory to the one with the downloaded files, and verify the signature with `gpg --verify Wasabi-${currentVersion}.deb.asc`.
+To verify the signature you also need to download the separate signature `.asc` file.
+In the terminal, change the directory to the one with the downloaded files, and verify the signature with the correct commands, depending on which operating system you are.
```
Comment (yahiheb) [inline docs/FAQ/FAQ-Installation.md:47]: IMO we should keep at least that command.

```suggestion
In the terminal, change the directory to the one with the downloaded files, and verify the signature with `gpg --verify Wasabi-${currentVersion}.deb.asc`, or with another command, depending on which operating system you are.
```

**Review by yahiheb — CHANGES_REQUESTED** [review]

File: docs/FAQ/FAQ-Installation.md (hunk 1 above)
Comment (MarnixCroes) [reply docs/FAQ/FAQ-Installation.md:47]: nack

Comment (yahiheb) [reply docs/FAQ/FAQ-Installation.md:47]: How would someone who does not know the command know which command to use?

Comment (MarnixCroes) [reply docs/FAQ/FAQ-Installation.md:47]: click on the links underneath for the full guide for each OS, where it is in more detail explained how to for mac, linux and windauws

Comment (yahiheb) [reply docs/FAQ/FAQ-Installation.md:47]: In the Windows and macOS there guides there is no mention of any command.
Plus this command works on all OSs.

Comment (MarnixCroes) [reply docs/FAQ/FAQ-Installation.md:47]: verify deb package on macOS or Win?

Comment (yahiheb) [reply docs/FAQ/FAQ-Installation.md:47]: Check the updated suggestion.

Comment (MarnixCroes) [reply docs/FAQ/FAQ-Installation.md:47]: > In the terminal, change the directory to the one with the downloaded files, and verify the signature with `gpg --verify` + the corresponding signature file name, depending on which operating system you are.

that's incorrect, you need both both the signature file and the package

Comment (yahiheb) [reply docs/FAQ/FAQ-Installation.md:47]: Both of these commands work:
`gpg --verify Wasabi-2.0.2.msi.asc`
`gpg --verify Wasabi-2.0.2.msi.asc Wasabi-2.0.2.msi`

Comment (MarnixCroes) [reply docs/FAQ/FAQ-Installation.md:47]: ok I didn't know that works on win and mac

**Review by MaxHillebrand — APPROVED** [review]

**Review by yahiheb — APPROVED** [review]

**Review by yahiheb — CHANGES_REQUESTED** [review]

**Review by yahiheb — APPROVED** [review]

**Pull request merged**